dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -114,6 +118,110 @@ def __rxor__(self, other):
         return _PipelineDebug.current()._wrap_op_call(_arithm_op, ["bitxor", other, self], {})
 
 
+def _transform_data_to_tensorlist(data, batch_size, layout=None):
+    data = _pipeline._prep_data_for_feed_input(data, batch_size, layout)
```

Comment:
Just checking, thanks to that the data is transformed to DALI types already, so we won't get here numpy for example?

Reply:
The idea here is to transform any data to TensorList (that includes [Tensor] -> TensorList). I've made it a separate function for the future implementation of the operator calls in the debug mode, where we would just use this function instead of an external source operator (supporting the debug mode feature of calling operators on data convertible to TensorList).